packages/xstate-react/test/useInterpret.test.tsx

```diff
@@ -260,4 +260,29 @@ describeEachReactMode('useInterpret (%s)', ({ suiteKey, render }) => {
 
     expect(childState.textContent).toBe('received');
   });
+
+  it('should deliver messages sent from an effect to the root actor registered in the system', () => {
```

Comment:
this one wasn't working because `useEffect(() => () => actor.stop())` was deleting the registered `systemId` and it was never re-registered